recipes/robotframework/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "robotframework" %}
+{% set version = "3.0.2" %}
+{% set sha256 = "2018f233568227f9af8c366f93e243a157b742f3baf5b649ecc9750ac0ee1ff7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: robotframework-3.0.2.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
```

Comment:
It looks like there are several command line tools that probably should be tested:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-commands

Reply:
Agreed, but the commands return errorlevel 251, so the tests fail. Even when running a -h command. So I leave them out.

Reply:
I'm a little concerned that the command line tools are failing as this might point to something not configured correctly. Can you provide the full traceback?

Reply:
I get this:
```
D:\> echo %errorlevel%
0

D:\> robot --version
c:\Conda3_64\envs\robo\lib\runpy.py:125: RuntimeWarning: 'robot.run' found in sys.modules after import of package 'robot', but prior to execution of 'robot.run'; this may result in unpredictable behaviour
  warn(RuntimeWarning(msg))
Robot Framework 3.0 (Python 3.5.3 on win32)

D:\> echo %errorlevel%
251
```
The same happens with the `--help` command.